### ISSUE TITLE
Version 0.3.27 - Home coming celebration task broken

### ISSUE THREAD

**Issue by pinkamedic** [loiste-interactive/Obenseuer-Issues#1574]
when going into the bosses penthouse the present will be empty but when leaving the building it will say that you have stolen the bottle of suca junior despite there being no bottle, after getting the poisoned suca junior recipe the present will be gone locking you from finishing the task

**Comment by Z-O-M-B-I-E** [comment]
A certain dialogue allowed me to complete this task. After some question, the gift box still appeared

**Comment by Zuntti** [comment]
Can I get a save where this happens?

**Comment by pinkamedic** [comment]
[Kyle_Pinkmin.zip](https://github.com/loiste-interactive/Obenseuer-Issues/files/13780486/Kyle_Pinkmin.zip)
save 1 has me standing infront of the empty box, if you leave talk to the guy then return it will be gone

**Comment by Orphiex** [comment]
Can confirm this issue. Entered Deekula C with the "Homecoming Celebration" task active, talked to the thug, made it to the coded door, and realized that I'd forgotten the passcode, so I went out to confirm it... and the "Steal the bottle (optional)" event triggered, despite not having even made it into the room.

So, I reloaded a fresh save, and went straight up (remembering the code this time); the gift box was on the desk, but the item slot was empty. The task marker still triggered when I left the apartment, despite not having a bottle of Socca Junior in my inventory.

I later found another bottle, poisoned it, and returned to Deekula C to deposit it in the gift box, but this time, the gift box was straight-up absent. With no place to put the Poisoned Socca Junior, that quest is softlocked indefinitely.
